xdmod-pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: xdmod-pv
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  Reclaim Policy: Retain
+  hostPath:
```

Comment:
We generally won't use `hostPath` volumes in our OpenShift deployments. They require elevated privileges, and they can't be accessed from different nodes like a regular PVC.

Reply:
This is how I am creating the PV on minikube.  As I don't need to worry about creating PVs on OpenShift.

Would it be better for me to keep the minicab deployment locally, or should it be under source control?

Reply:
I didn't realize you were explicitly targeting minikube right now. If that's the case, I guess leave these here for now and we'll revisit when we get around to deploying in openshift.

Reply:
I am planning on both.  minikube is more convenient for development.

I am keeping it separated, that the deployment to openshift under /k8s/..., and the pv definitions specifically for minikube out of that directory for now.